clientv3/watch.go

```diff
@@ -52,6 +59,22 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsCreateEvent returns true if the event tells that the key is newly created.
+func IsCreateEvent(e *Event) bool {
```

Comment:
as you are already on the testing thing, can you also write a simple unit test for these two helper func? Just feed in event and compare the bool output?